pkg/operator/kuryr.go

```diff
@@ -0,0 +1,147 @@
+package operator
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+
+	netv1 "github.com/openshift/api/network/v1"
+	"github.com/operator-framework/operator-sdk/pkg/util/k8sutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	uns "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/openshift/cluster-network-operator/pkg/render"
+)
+
+// renderKuryr returns manifests for Kuryr SDN.
+// This creates
+// - the ClusterNetwork object
+// - the kuryr namespace
+// - the kuryr-controller deployment
+// - the kuryr-daemon daemonset
+// and some other small things.
+func (h *Handler) renderKuryr() ([]*uns.Unstructured, error) {
+	operConfig := h.config.Spec
+	c := operConfig.DefaultNetwork.KuryrConfig
+
+	objs := []*uns.Unstructured{}
+
+	// generate master network configuration
+	ippools := []netv1.ClusterNetworkEntry{}
+	for _, net := range operConfig.ClusterNetworks {
+		ippools = append(ippools, netv1.ClusterNetworkEntry{CIDR: net.CIDR, HostSubnetLength: net.HostSubnetLength})
+	}
+
+	clusterNet := netv1.ClusterNetwork{
```

Comment:
a ClusterNetwork is just used for `openshift-sdn` - do you need to create it? I don't think so.